ucsb-cs156-s24/team03-s24-5pm-6
# Anika - Created a Create page for Recommendation Request

Test coverage 100, all tests pass, changed some directory names from ReccomendationRequest to RecommendationRequests
Link to storybook entry for Create page: http://localhost:6006/?path=/story/pages-recommendationrequest-recommendationrequestcreatepage--default

Steps to test: 
Click RecommendationRequest on navbar and press Create
![image](https://github.com/ucsb-cs156-s24/team03-s24-5pm-6/assets/91992504/189c0601-a3b1-48ad-8bfa-2239dbd4ead7)

Fill out entries accordingly
![image](https://github.com/ucsb-cs156-s24/team03-s24-5pm-6/assets/91992504/806781cf-d035-4e75-9a73-da43587877c2)

There should be a message saying the entry was created successfully if it is valid: 
![image](https://github.com/ucsb-cs156-s24/team03-s24-5pm-6/assets/91992504/2092d00a-213b-47ba-9f86-90fffdac3525)

If you then navigate to Swagger, and list all recommendation requests, if your entry was valid you should see it listed: 
![image](https://github.com/ucsb-cs156-s24/team03-s24-5pm-6/assets/91992504/d430ab51-2c22-4e1e-9e05-d3ab5fffda11)

Closes issue #31
